redisearch/client.go

```diff
@@ -148,6 +148,16 @@ type IndexingOptions struct {
 	Replace  bool
 }
 
+// MultiError Represents one or more errors
+type MultiError map[int]error
```

Comment:
I'd rather have a slice here where each sub error correlates to the index of the operation that triggered it